source/extensions/stat_sinks/common/statsd/statsd.h

```diff
@@ -65,6 +70,8 @@ class UdpStatsdSink : public Stats::Sink {
   ThreadLocal::SlotPtr tls_;
   Network::Address::InstanceConstSharedPtr server_address_;
   const bool use_tag_;
+  // Prefix for all flushed stats.
+  std::string prefix;
```

Comment:
`const std::string prefix_`. Just initialize directly in the initializer list using the ternary operator.

Reply:
Done!